README.md

```diff
@@ -1,2 +1,69 @@
 # fastify-get-head
-Fastify plugin to set a HEAD route handler for each GET handler previously registered
+
+[![Greenkeeper badge](https://badges.greenkeeper.io/fastify/fastify-cookie.svg)](https://greenkeeper.io/)
+
+<!-- [![Build Status](https://travis-ci.org/fastify/fastify-cookie.svg?branch=master)](https://travis-ci.org/fastify/fastify-cookie) -->
+
+A plugin for [Fastify](http://fastify.io/) that adds support for setting a `HEAD` route for each `GET` one previously registered.
+
+This plugin's cookie parsing works via Fastify's `onRoute` hook. When a new route is registered, the plugin will try to set a new `HEAD` route if the registered route is for a `GET` method.
+
+`fastify-cookie` v3 only supports Fastify@3.x.
+
+<!-- Setted once https://github.com/fastify/fastify/commit/449fc5c25e7d2e277e68bb3f4416cfd858a71cec is released -->
+
+## Example
+
+```js
+const fastify = require('fastify')();
+
+fastify.register(require('fastify-get-head'), {
+  ignorePaths: ['/api/ignore', /\/api\/ignore\/too/], // For ignoring specific paths
```

Comment:
Does this support regexps too?

Reply:
Yes it does, I added support for regexps and string, so it's possible to use either or both of them. See [here](https://github.com/MetCoder95/fastify-get-head/pull/1/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R3)